examples/nlp_si_detection/README.md

```diff
@@ -116,16 +116,14 @@ morpheus --debug --log_level=DEBUG \
    `# 3rd Stage: Preprocessing converts the input data into BERT tokens` \
    preprocess --vocab_hash_file=$MORPHEUS_ROOT/morpheus/data/bert-base-uncased-hash.txt --do_lower_case=True --truncation=True \
    `# 4th Stage: Send messages to Triton for inference. Specify the model loaded in Setup` \
-   inf-triton --model_name=sid-minibert-onnx --server_url=localhost:8001 --force_convert_inputs=True \
+   inf-triton --model_name=sid-minibert-onnx --server_url=localhost:8000 --force_convert_inputs=True \
```

Comment:
Will this always be using the HTTP bindings? Using the `gRPC` port of 8001 will fall back to 8000 if it doesnt connect but the reverse wont happen. So its more of a risk to use 8000 unless we 100% know we will always be using the C++ version of this stage.

Reply:
Currently using the grpc port will trigger this warning:
```
W20220627 10:48:47.255275 2432276 triton_inference.cpp:248] Failed to connect to Triton at 'localhost:8001'. Default gRPC port of (8001) was detected but C++ InferenceClientStage uses HTTP protocol. Retrying with default HTTP port (8000)
```

So I switched the port so that the example will run without any errors

Reply:
Ok